packages/api/src/router/invite.ts

```diff
@@ -0,0 +1,70 @@
+import { randomBytes } from "crypto";
+import { TRPCError } from "@trpc/server";
+
+import { asc, createId, eq } from "@homarr/db";
+import { invites } from "@homarr/db/schema/sqlite";
+import { z } from "@homarr/validation";
+
+import { createTRPCRouter, protectedProcedure } from "../trpc";
+
+export const inviteRouter = createTRPCRouter({
+  all: protectedProcedure.query(async ({ ctx }) => {
+    const dbInvites = await ctx.db.query.invites.findMany({
+      orderBy: asc(invites.expirationDate),
+      columns: {
+        token: false,
+      },
+      with: {
+        creator: {
+          columns: {
+            id: true,
+            name: true,
+          },
+        },
+      },
+    });
+    return dbInvites;
+  }),
```

Comment:
Aren't we missing the "byId" procedure?
Or is the check done by going against all existing invites?

Reply:
It's not yet implemented that you can join Homarr with an invite url. This is only the management of it